resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ResourceMethodInvoker.java

```diff
@@ -335,14 +338,15 @@ public BuiltResponse invoke(HttpRequest request, HttpResponse response, Object t
       }
       uriInfo.pushCurrentResource(target);
       BuiltResponse rtn = invokeOnTarget(request, response, target);
-      return rtn;
+      // FIXME: async

```

Comment:
?

Reply:
Doesn't look like something I should fix, I'll remove it.